feel/feelmodels/modelmaterials.hpp

```diff
@@ -460,7 +485,7 @@ material( ModelMaterials::value_type const& m )
 }
 
 FEELPP_EXPORT inline std::string
-marker( ModelMaterials::value_type const& m )
+name( ModelMaterials::value_type const& m )
```

Comment:
won't it break some code to rename such a function?

Reply:
I think you should duplicate the function, rename it and `FEELPP_DEPRECATED` `marker`  if you want to do that